modules/special-rules/pages/heavy-barrage.adoc

```diff
@@ -1,18 +1,20 @@
 = Heavy Barrage
 
 These weapons are like {barrage} weapons, but even more destructive.
-They are also cumbersome and slow to reload -- only war engines are big enough to maintain a constant feed of ammunition to such a weapon during a battle. 
+They are also cumbersome and slow to reload -- only war engines are big enough to maintain a constant feed of ammunition to such a weapon during a battle.
 
 ---
 
-When you shoot with a {heavy-barrage} weapon, proceed exactly as for a {barrage} weapon but with these differences:
+When you make a {heavy-barrage} shot, proceed exactly as for a {barrage} shot but with these differences:
 
-* The Firepower value is equal to *twice* the number of units under the Barrage template.
+* The Firepower value is equal to *twice* the number of units that are under the Barrage template.
 * If the weapon is on a War Engine unit then you may continue to move and shoot with this unit each turn as normal.
 Otherwise, you may not *move* or *shoot* again with the unit until it has had a full turn to reload.
 Turn the model around, mark it with a token or make a note so that you don't forget that it must reload before you can use it again.
 +
-+[TODO: Clarify what should happen if the unit is forced to move, as in a withdrawal. Presumably they still can't move and so they are automatically destroyed if the enemy gets close enough.]+
+{blank}[TODO: Clarify what should happen if the unit is forced to move, as when their detachment is Broken and makes a Retreat move.
+Option *A*: They can't move as stated above, and so in the retreat situation they are automatically destroyed if the enemy gets close enough.
+Option *B*: When the move is compulsory, as in the retreat situation, then they can move -- in contradiction to the above rule -- but they can't reload until after they've had a full turn with no movement.]
```

Comment:
TODO note to be removed.

```suggestion

```

Reply:
I don't think that we should just remove it — I put it there for a reason that we haven't resolved.

I suggest either we leave it here to tackle at some future time after we publish this PR, or we agree on one of the options and write it in. Otherwise, we just leave the question hanging but not noted for future attention.

Given your note on the main thread:

>looking at the units it would affect (Chaos Artillery, Bombards/Manticores) I don't have an issue with using the text as-is, and that the unit is effectively immobilised.

...I think that in effect you're saying that you favour option A...? That is also my preference, as I feel that it is simpler and more consistent than option B.

Reply:
> ...I think that in effect you're saying that you favour option A...? That is also my preference, as I feel that it is simpler and more consistent than option B.

Not quite. I think that option B requires changing the rule, and I withdraw my original suggestion that you can 'delay' reloading. As written, the unit cannot move or shoot in the following turn, as the TODO note points out.

See https://github.com/GameBrains/er-core/blob/c6edd687b0fba282043e22763769af0f86669396/modules/special-rules/pages/heavy-barrage.adoc for the older version.

I think the main text should be OK, but if you think it isn't clear enough, I don't think much change would be needed:

Current: Otherwise, you may not move or shoot again with the unit until it has had a full turn to reload.
Suggestion: Otherwise, you may not move or shoot with the unit in the following turn.

Reply:
> Suggestion: Otherwise, you may not move or shoot with the unit in the following turn.

I don't see how this answers the question: What happens when something that 'may not move or shoot' at the time is 'forced to move' (as in a retreat move)...? Do we override 'may not move' (option B), or do we not (option A)?

Reply:
> I don't see how this answers the question: What happens when something that 'may not move or shoot' at the time is 'forced to move' (as in a retreat move)...? Do we override 'may not move' (option B), or do we not (option A)?

Ah, I think I see where we're talking past each other! 

When a detachment becomes broken it 'must retreat', but I can't find any requirement to actually move - just a penalty if the unit doesn't end up 15 cm from all enemies. So a reloading Heavy Barrage unit would stay in place and die for ending within 15 cm. This is almost identical to a unit  that fails the first part of a Dangerous terrain test and can't move away.

The only actual forced move that I can think of is for Assault moves and we know that's overridden by not being able to move, whether that's from a failed Leadership test or Dangerous terrain test.

So I don't see any issue with Heavy Barrage in particular.

Reply:
Lol, okay, we're on the same page now. Arguably, given those other precedents, players should realise that the Retreat move does not override 'may not move' — but I feel that it is good to spell it out clearly so that they know that this isn't an 'override' situation.

I think something like this should suffice:

```suggestion
NOTE: To be clear, if your {heavy-barrage} units are in the process of reloading while their detachment becomes Broken, they cannot join in any Retreat move that you make.
Thus they are more likely to be caught and destroyed by the enemy.
```

Reply:
OK, but I'd expect to see text for "players should realise that the Retreat move does not override 'may not move'" in the Retreat rule, not in the individual rules that stops you moving.

Reply:
Hmmm. I see what you mean. I'll edit `broken-detachments.adoc`. But that throws up another question — where at present it says this:

> The Retreat move is a single move in which you may move all of the Broken units up to 20 cm -- regardless of the actual Speed values of units.

...do you think that we should amend this slightly to cover the possibility that we might have units with a Speed value of  `0 cm` or `-` that are effectively always units that 'may not move'?

The answer to this affects how best to word the 'does not override' bit.

Reply:
This is now outdated as I've removed the TODO, and edited the other topic as you suggest — so I'll resolve this thread. Continues in the new context at https://github.com/GameBrains/er-core/pull/14/files#r684422983.